scripts/apps/desks/views/desk-config-modal.html

```diff
@@ -151,25 +151,50 @@ <h3 class="modal__heading" ng-show="desk.edit._id" translate>Edit "{{desk.edit.n
                         </div>
                     </div>
                     <div class="form__row">
-                        <div class="sd-line-input sd-line-input--is-select">
+                        <div class="sd-line-input sd-line-input--required">
                             <label class="sd-line-input__label" translate>Default Content Template</label>
-                            <select
-                                ng-model="desk.edit.default_content_template"
-                                ng-options="template._id as template.template_name for template in templates"
-                                class="sd-line-input__select"
-                                data-test-id="field--default-content-template"
-                            >
+                            <div class="sd-alert sd-alert--hollow sd-alert--small">
+                                <i class="icon-info-sign icon--blue icon--2x sd-alert__icon"></i>
+                               <span translate>
+                                   Select a default template for new articles. If you don't see any, first make sure that there is at least one content profile. Each time a profile is created, Superdesk creates a template with the same name.
+                                </span>
+                            </div>
+                            <div class="sd-line-input sd-line-input--is-select">
+                                <select
+                                    ng-model="desk.edit.default_content_template"
+                                    class="sd-line-input__select"
+                                    required
+                                    data-test-id="field--default-content-template"
+                                >
                                 <option value=""></option>
-                            </select>
+                                <option ng-repeat="template in templates" value="{{template._id}}" data-test-id="template {{template.template_name}}" ng-selected="desk.edit.default_content_template === template._id">
```

Comment:
You don't need `data-test-id` here. It's value is template name with a prefix, so we can instead add `data-test-id` on a `select` itself and then select by option value without having to add ids for every option.

`el(['field--default-content-template'], by.cssContainingText('option', 'template 5')).click();`